package.json

```
@@ -63,7 +63,7 @@
     "typescript": "4.1.5"
   },
   "peerDependencies": {
-    "@capacitor/core": "^4.0.0"
+    "@capacitor/core": "5.0.0"
```

```suggestion
    "@capacitor/core": "^5.0.0"
```

Only this line must remain `^5.0.0`, otherwise you cannot install the plugin with Capacitor version `5.0.1` etc.